src/db.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+function connectToDb (string $dbName): PDO {
+    $db = new PDO ('mysql:host=db;dbname=' . $dbName , 'root', 'password');
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    return $db;
+}
+
+function getAllDeities (PDO $db): array {
```

Comment:
on lines 3, 4, 9 - there should not be a space between the function and the (), e.g. `getAllDeities(PDO $db)`